scripts/unit/Makefile

```diff
@@ -0,0 +1,5 @@
+.DEFAULT_GOAL := tests_catapult.sh
+
+.PHONY: tests-catapult
```

Comment:
`unit` is too much scoped imho, we might end up to put also some integration tests at some point. catapult tests problably should be outside of `scripts/` and just be a top level folder